server/chat-commands/admin.ts

```diff
@@ -943,11 +950,29 @@ export const commands: ChatCommands = {
 			} else {
 				result = Utils.visualize(result);
 			}
-			result = result.replace(/\n/g, '\n||');
-			this.sendReply('||<< ' + result);
+			result = show && result.length > 200 ? (`<details class="readmore code" style="` +
+				Utils.html`display: table; tab-size: 3"><summary>${result.slice(0, 200)}</summary>` +
+				Utils.html`${result.slice(200)}</details>`
+			).replace(/\n/g, '<br>') : result.replace(/\n/g, '\n||');
```

Comment:
I suspect we should probably have a `Chat` function for this (I also want to do this for inputs as well as outputs).

Reply:
Also it shouldn't be at 200 characters, it should be at the first line break.

(I'll make a separate commit to cut make `Utils.visualize` puts line breaks in the right places.)